Tests/FeaturevisorSDKTests/ConditionsTests.swift

```diff
@@ -0,0 +1,287 @@
+import XCTest
+
+@testable import FeaturevisorSDK
+@testable import FeaturevisorTypes
+
+class ConditionsTests: XCTestCase {
+
+    func testBooleanEquals() {
+        let condition = PlainCondition(
+            attribute: "isTrue",
+            operator: .equals,
+            value: .boolean(true)
+        )
+        var context = Context()
+        context["isTrue"] = .boolean(true)
+
+        XCTAssertTrue(conditionIsMatched(condition: condition, context: context))
+    }
+
+    func testBooleanNotEquals() {
```

Comment:
Follow the same pattern that we have in other tests. Point out sections like GIVEN, WHEN, THEN